pkg/reconciler/persistentvolumeclaim/requests.go

```diff
@@ -56,8 +56,9 @@ func reconcilePVCQuantity(
 ) error {
 	contextLogger := log.FromContext(ctx)
 	pvcRole := utils.PVCRole(pvc.Labels[utils.PvcRoleLabelName])
+	tablespace := pvc.Labels[utils.PvcTablespaceLabelName]
```

Comment:
NOTE: this is TBD. This is assuming that the pvc was labeled with the tablespace name, and with that we can map back to the tablespace definition in the cluster spec.
Tao points out that since the PVC name is normalized `clustername-serial-tbs-tablespacename` we could parse the tablespace from the PVC name. Worth discussing.